website/docs/cloud-docs/workspaces/settings/deletion.mdx

```diff
@@ -25,31 +25,60 @@ Before queuing a destroy plan, enable the **Allow destroy plans** toggle setting
+* Schedule a run to destroy all resources in a workspace at a specific date and time.
+* Configure Terraform Cloud to destroy a workspace's infrastructure after a period of workspace inactivity.
+
+You can reduce your spending on infrastructure by automatically destroying temporary resources like development environments.
+
+After Terraform Cloud performs an auto-destroy run, it unsets the `auto-destroy-at` field on the workspace. If you continue using the workspace, you can schedule another future auto-destroy run to remove any new resources. 
 
 !> **Note:** Automatic destroy plans _do not_ prompt you for apply approval in the Terraform Cloud user interface. We recommend only using this setting for development environments.
 
-You can schedule an auto destroy run using the Terraform Cloud web UI, or the [workspace API](/terraform/cloud-docs/api-docs/workspaces).
+You can schedule an auto-destroy run using the Terraform Cloud web user interface, or the [workspace API](/terraform/cloud-docs/api-docs/workspaces).
+
+You can schedule [notifications](/terraform/cloud-docs/workspaces/settings/notifications) to alert you 12 and 24 hours before an auto-destroy run, and to report auto-destroy run results.
+
+#### Destroy at a specific day and time
 
-To schedule an auto destroy run in Terraform Cloud:
+To schedule an auto-destroy run at a specific time in Terraform Cloud:
 1. Navigate to the workspace's **Settings** > **Destruction and Deletion** page.
 1. Under **Automatically destroy**, click **Set up auto destroy**.
 1. Enter the desired date and time. Terraform Cloud defaults to your local time zone for scheduling and displays how long until the scheduled operation.
 1. Click **Confirm auto destroy**.
 
-To cancel a scheduled auto destroy run in Terraform Cloud:
+To cancel a scheduled auto-destroy run in Terraform Cloud:
 1. Navigate to the workspace's **Settings** > **Destruction and Deletion** page.
 1. Under **Automatically destroy**, click **Edit** next to your scheduled run's details.
 1. Click **Remove**.
 
-You can schedule [notifications](/terraform/cloud-docs/workspaces/settings/notifications) to alert you 12 and 24 hours before an auto destroy run, and to report auto destroy run results.
+#### Destroy if a workspace is inactive
 
-[execution mode](/terraform/cloud-docs/workspaces/settings#execution-mode) 
+You can configure Terraform Cloud to automatically destroy a workspace's infrastructure after a period of inactivity.
+A workspace is _inactive_ if the workspace's state has not changed within your designated time period.
+
+!> **Note:** As opposed to configuring an auto-destroy run for a specific date and time, this setting will _persist_ after queued auto-destroy runs.
+
+If you configure a workspace to auto-destroy its infrastructure when inactive, any run that updates Terraform state further delays the scheduled auto-destroy time by the length of your designated time period.
```

Comment:
Suggesting changing this to "timeframe" because that's what the UI uses:
```suggestion
If you configure a workspace to auto-destroy its infrastructure when inactive, any run that updates Terraform state further delays the scheduled auto-destroy time by the length of your designated timeframe.
```